client/packages/common/src/hooks/useFeatureFlags/useFeatureFlags.ts

```diff
@@ -36,5 +36,7 @@ export const useFeatureFlags = () => {
     // Enable the new demo create stocktake modal (see https://github.com/msupply-foundation/open-msupply/issues/8773)
     createStocktakeModalUsabilityImprovements:
       !!featureFlags['create_stocktake_modal_usability_improvements'],
+    // Enable the new demo table, with better usability
+    tableUsabilityImprovements: !!featureFlags['table_usability_improvements'],
```

Comment:
Conflicts here, is this supposed to be removed in 2.12?

Reply:
I think it's supposed to be removed? @CarlosNX ?

Reply:
Both can go! I think conflicts as `createStocktake` one removed in 2.11, table usability one removed in develop 🙏